docs/APIReference-Editor.md

```diff
@@ -106,6 +106,31 @@ for details on usage.
 
 ### Behavior (Optional)
 
+### autoCapitalize?: string
+
+```
+autoCapitalize?: string
+```
+
+Set if auto capitalization is turned on and how it behaves. More about platform availability and usage can [be found on mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#attr-autocapitalize).
```

Comment:
@flarnie the thing is - with `spellCheck` it is just a boolean so you can set it on or off.

But with `autocapitalize` you have several options:

`none`
`sentences`
`words`
`characters`
`on`
`off`

So just saying you can turn it off and on doesn't cut it for me. I would include the options you can give it, but I'm afraid that the docs would go out of date fairly quickly since it's such a new nonstandard feature...